tex/generic/pgf/modules/pgfmoduleparser.code.tex

```diff
@@ -1,4 +1,4 @@
-% Copyright 2009 by Till Tantau
```

Comment:
Don't remove copyright statements of other people.

Reply:
I removed it because it doesn't share a single line (and if so that is by accident). In the documentation Till's copyright isn't removed because I used almost all of his text. If removing it under these conditions was false, I apologize, it wasn't my intention to steal any interlectual property.

Reply:
Your parser is based on Till's original version, making it a derived work.  The GPL requires that derived works contain the same copyright notices as the original.

But apart from the legal aspect, I think Till deserves some credit.  Your parser would probably not exist without his version.

Reply:
Yes, that is correct. Though my parser is really only based on the documentation in the manual, not on his code. Still your point is valid.